### ISSUE TITLE
Fix exception on maps link

### ISSUE THREAD

**Issue by DeD1rk** [svthalia/Reaxit#172]
### Describe the bug
Sadly the android maps link on the EventScreen does seem to throw in some cases

The problem seems to be here (in the url_launcher plugin):
<img width="1173" alt="image" src="https://user-images.githubusercontent.com/41264528/145394356-4c0fd781-c4b6-40f9-a524-3156e27fc24b.png">

canLaunch returns false. That seems to be because of this:
<img width="1291" alt="image" src="https://user-images.githubusercontent.com/41264528/145394661-b86192e5-e562-4be3-aea6-0426c67d93c2.png">

Basically, a permission needs to be declared in the androidmanifest to allow querying whether an app is available to handle some url scheme. This seems rather useless for https links, so perhaps the best solution would be to simply remove the Link, and call `launch(<the correct uri and the correct target settings>)` directly.

**Comment by DeD1rk** [comment]
The same thing seems to be happening for all other Links. I would say this is a bug in url_launcher (will create/upvote issue), but we should wrok around it anyway.
